specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2021-07-01-preview/galleryImages.json

```diff
@@ -0,0 +1,439 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AzureStackHCI",
+    "description": "The Microsoft.AzureStackHCI Rest API spec.",
+    "version": "2021-07-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureStackHCI/galleryimages/{galleryimagesName}": {
+      "get": {
+        "operationId": "galleryimages_Retrieve",
+        "description": "Lists all galleryimages under the resource group.",
+        "x-ms-examples": {
+          "GetGalleryImage": {
+            "$ref": "./examples/GetGalleryImage.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v2/types.json [](http://example.com/codeflow?start=69&length=13)

Consider moving to v3/types.json. Comment applies throughout this PR.

Reply:
Added this workitem to this pending task: https://dev.azure.com/msazure/msk8s/_boards/board/t/ARC-HCI/Stories/?workitem=12679581